README.md

```diff
@@ -63,12 +63,15 @@ You need [CMake](https://cmake.org/download/) to configure the Exiv2 project and
 
 <div id="2-1">
 
-### 2.1 Build, Install, Use, Uninstall Exiv2 on a UNIX-like system
+### 2.1 Build, Install, Use Exiv2 on a UNIX-like system
 
 ```bash
-$ cd <exiv2dir> ; mkdir build ; cd build
-$ cmake .. -G "Unix Makefiles" ; cmake --build . ; make test
-$ sudo make install
+cd $EXIV_ROOT
+mkdir build && cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
+cmake --build .
+make test
+sudo make install
```

Comment:
I wouldn't recommend running that since we do not provide an official uninstallation method.